homeassistant/components/lookin/climate.py

```diff
@@ -115,6 +116,7 @@ async def _async_update() -> Climate:
 class ConditionerEntity(LookinCoordinatorEntity, ClimateEntity):
     """An aircon or heat pump."""
 
+    _attr_current_humidity: float | None = None  # type: ignore
```

Comment:
This is indeed typed incorrectly in the main codebase

Reply:
https://github.com/home-assistant/core/pull/61456

Reply:
We can clean this up later in a future PR once #61456 is discussed and resolved

Reply:
i changed the value to int